packages/formstr-app/src/App.js

```diff
@@ -3,14 +3,23 @@ import "./App.css";
 import Routing from "./components/Routing";
 import { ProfileProvider } from "./provider/ProfileProvider";
 import { ApplicationProvider } from "./provider/ApplicationProvider";
+import { UserProvider } from "./provider/UserProvider";
+import { ListProvider } from "./provider/ListProvider";
```

Comment:
i can see a use case for this, but we should not write new User and App Providers when there are already existing ones.